config/routes/aytq.rb

```diff
@@ -6,17 +6,18 @@
 get "/sign-in", to: "users/sign_in#new"
 get "/sign-out", to: "users/sign_out#new"
```

Comment:
We could move sign-in / sign-out into the namespace, and update the controller namespace for the callback. Shouldn't affect Identity integration?